MAINTAINERS.yml

```diff
@@ -3184,7 +3184,7 @@ West:
   files:
     - modules/Kconfig.intel
   labels:
-    - manifest-hal_intel
+    - "platform: Intel"
```

Comment:
This label does not exist, can you please make sure all those labels actually exist and they corrospond to something we already use in the Maintainer file?

Reply:
I created the label while preparing the PR